src/main.cpp

```diff
@@ -16,6 +18,8 @@ int main(int argc, char *argv[])
         qApp->setFont(QFont("Ubuntu", 11, QFont::Normal, false));
     #endif
 
+	std::srand(std::time(nullptr));
```

Comment:
This is not cryptographically secure.

Reply:
Fixed in b150f074f7ad2baad3179510b5f716cc1ad79bb6